docker-compose-development.override.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
```

Comment:
I'm actually thinking we might want to skip the .env file and just hard-code the ports here.  It seems like an extra step that's probably not needed, since this file is for local environments only.